lively.ide/js/browser/index.js

```diff
@@ -1125,13 +1139,23 @@ export class BrowserModel extends ViewModel {
     }
 
     if (['directory', 'package'].includes(selectedFile.type)) {
-      this.ui.sourceEditor.opacity = 0.7;
-      this.ui.sourceEditor.readOnly = true;
-      this.updateSource('');
-      this.ui.metaInfoText.showInactive();
+      this.deactivateEditor();
     }
   }
 
+  deactivateEditor () {
+    this.ui.sourceEditor.opacity = 0.7;
+    this.ui.sourceEditor.readOnly = true;
+    this.updateSource('');
+    this.ui.metaInfoText.showInactive();
+  }
+
+  reactivateEditor () {
+    this.ui.sourceEditor.opacity = 1;
+    this.ui.sourceEditor.readOnly = false;
+    this.ui.metaInfoText.showInactive();
```

Comment:
Just a nitpick, but same as earlier, I prefer a top level: `const { sourceEditor, metaInfoText } = this.ui;` instead of having countless `this.ui` refs in the code.